airbyte-webapp/src/pages/connections/CreateConnectionPage/CreateConnectionPage.tsx

```diff
@@ -238,6 +200,7 @@ export const CreateConnectionPage: React.FC = () => {
               }
             />
           )}
+          {currentStep !== StepsTypes.CREATE_CONNECTION && <InlineEnrollmentCallout withBottomMargin />}
```

Comment:
This in an inversion of the condition a few lines up in https://github.com/airbytehq/airbyte/blob/340443cbb9f00e7f85e678326dc8f124f422be1b/airbyte-webapp/src/pages/connections/CreateConnectionPage/CreateConnectionPage.tsx#L189
That `big` prop removes the containing element's side margin; since the UI below it is spaced via an internal padding rule, not its external context, the `InlineEnrollmentCallout` doesn't line up.

This fix is undeniably ugly, but it struck me as the least of the available evils in terms of maintenance burden:
- conditionally adding a class to add a `margin: 0 variables.$spacing-xl` adds this implicit, invisible dependency on an implementation detail of `<FormPageContent big />`
- adding a bottom margin prop to `InlineEnrollmentCallout` does too, but the coupling is even further removed from its source
- it would be nice to show the enrollment banner here to, but by this step in the connection creation process, the user has already seen it at least once (very likely twice) on the "select source"/"select destination" subpage(s), and the very next step takes them to the new connection page--which also has an enrollment banner.

Okay, there's one option that's a whole lot nicer: rewrite large swaths of the create connection UI to use consistent styling techniques. I opted not to go down that potential rabbit hole as part of this PR though.

Reply:
Rewrite (+1)